submissions/xT0PERx/html-movie-seat-booking/main.css

```diff
@@ -0,0 +1,357 @@
+html {
+  display: grid;
+  justify-content: center;
+}
+
+.page {
+  max-width: 1550px;
+  background: rgb(201, 201, 214);
+}
+
+.top {
+  width: 100%;
+  background: linear-gradient(
+    90deg,
+    rgba(119, 109, 106, 1) 1%,
+    rgba(201, 201, 214, 1) 100%
+  );
+  border-radius: 10px;
+}
+
+.navigation {
+  margin-left: 0.5em;
+  margin-right: 0.5em;
+}
+
+.items {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 20px;
+  list-style-type: none;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  padding-inline-start: 0em;
+}
+
+.item {
+  display: grid;
+  width: 100%;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  align-self: center;
+  justify-self: center;
```

Comment:
You can use `place-items: center` for centering vertically and horizontally.

Reply:
Perhaps this is my misuse of "place-items: center". But I was unable to get the exact behavior of my elements. Tried on classes: .navigation_item, .screen. The elements aligned but started jumping - is there any point in bothering to prescribe border: transparent to them?
Worked on .btn_prevDay, .btn_nextDay classes - but only replaces the "justify-content: center" property. I don't know if this makes sense?
It worked as expected only on ".seat_number". Perhaps I’m missing something. I’ll be glad to your advice. Thank you.